packages/core/src/main.ts

```diff
@@ -45,7 +42,7 @@ export {
 
     AutoscaledPool,
 
-    BasicCrawler,
+    // BasicCrawler,
```

Comment:
i see - this should definitely go, we should not export things from here that we are moving to their own packages, all the crawler related exports should go, and we should import them from the basic crawler package instead (and as said on slack, going forward we should split that and change imports to the leaf packages).

Reply:
Honestly.. the exports in core are a mess, we should really clean them up completely. Is this file or exports.ts even needed anymore?